pkg/osquery/runtime/runner.go

```diff
@@ -656,3 +660,62 @@ func newRunner(opts ...OsqueryInstanceOption) *Runner {
 		opts:     opts,
 	}
 }
+
+func (r *Runner) runOsqueryVersionCheckAddToKnapsack(ctx context.Context, osquerydPath string) {
```

Comment:
Could we add a little bit of documentation here? `// runOsqueryVersionCheckAddToKnapsack checks the version of osquery at the given osquerydPath and saves it to the knapsack, so that it is available to add when logshipping` or similar?

Reply:
added documentation